prdoc/pr_1445.prdoc

```diff
@@ -0,0 +1,19 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: removed `without_storage_info` from `pallet-collective`
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      1. Voting storage is now a CountedStorageMap 
+      2. Members storage is now bounded, avoiding runtime checks
```

Comment:
> avoiding runtime checks

I guess the runtime checks still happen, but they're just behind the scenes now instead of explicit?